src/components/Button/index.js

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+import { Alert, Button } from 'react-native';
+import { ContainerButton } from './styles';
+
+export default function ButtonMain({ children }) {
+  return (
+    <ContainerButton>
+      <Button
+        color="#000000"
+        title={children}
+        onPress={() => Alert.alert('Works')}
```

Comment:
O onPress é bom receber de parametro
```suggestion
        onPress={() => Alert.alert('Works')}
```